src/client-apps/nuxeo-drive/nuxeo-drive-faq/nuxeo-drive-architecture-recommendations-windows.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Nuxeo Drive Configuration Requirements on Windows
+review:
+  comment: ''
+  date: '2024-10-05'
+  status: ok
+labels:
+  - nuxeo-drive
+  - mlumeau
+  - multiexcerpt
+  - excerpt
+  - lts2023-ok
```

Comment:
We can also specify the Nuxeo Drive version for which this requirements is recommended